Add 1.7.0 post (#393)
runelite/runelite.net: src/_posts/2021-02-13-00-00-1.7.0-Release.md

While we still do not have an MacOS launcher that runs native on the Apple M1, the core client has been
updated so it can run natively if you run the launcher `RuneLite.jar` [directly with a native JVM](<https://github.com/runelite/runelite/wiki/MacOS-Arm64-(M1-chips)-Native-Support>).
If you are technically apt enough to do this, it may be worth it if you have one of these new machines.
We still plan on providing a native launcher for the M1 in the future.

# Farming tracker notifications

The time tracking plugin now has an option to send notifications when crops are done growing.
It can be enabled per-crop by clicking the bell icon.

![Farming Notification](/img/blog/1.7.0-Release/farming_notif.png)

There are also several smaller improvements and bug fixes, including:

- The tag tab named `clue` now works once again
- The entity hider now allows hiding ignored players
- Chat messages replayed by the chat history plugin on login no longer re-trip commands or notifications.
  They also no longer lose their coloring.
- The fishing plugin can now send notifications for flying fish when fishing minnows
- The idle notifier now works at the Ectofuntus and at looms

> - Spoon and weary face emojis have been addded (`--o` and `Dx` respectively)

Reviewer: ```suggestion
- Spoon and weary face emojis have been added (`--o` and `Dx` respectively)
```

> - The loottracker now tracks the new chests from the Shades of Morton and Isle of Souls

Reviewer: ```suggestion
- The loot tracker now tracks the new chests from the Shades of Morton and Isle of Souls
```

- The chat notifications plugin has a new option to send you notifications for receiving private messages

> - The despawn timer in the ground items plugin was fixed for the Nightmare instance, and one has been added for tables

Reviewer 2: ```suggestion
- The despawn timer in the ground items plugin was fixed for the Nightmare instance, and one has been added for items placed on tables
```

> - A `Use` swap was added for battlestaffs

Reviewer 3: ```suggestion
- A `Use` swap has been added for battlestaves
```
